contracts/LiquidityLocker.sol

```diff
@@ -1,12 +1,17 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 pragma solidity 0.6.11;
 
-import { IERC20, SafeERC20 } from  "../../../../lib/openzeppelin-contracts/contracts/token/ERC20/SafeERC20.sol";
-
-import { ILoan } from "../../loan/contracts/interfaces/ILoan.sol";
+import { IERC20, SafeERC20 } from  "../modules/openzeppelin-contracts/contracts/token/ERC20/SafeERC20.sol";
 
 import { ILiquidityLocker } from "./interfaces/ILiquidityLocker.sol";
 
+// NOTE: IFundableLoan exists to prevent circular dependency tree.
+interface IFundableLoan {
+
+    function fundLoan(address debtLocker, uint256 amount) external;
+
+}
```

Comment:
What do we think about doing this for one line interfaces?
`interface IFundableLoan { function fundLoan(address debtLocker, uint256 amount) external; }`